docs/troubleshooting.md

```diff
@@ -3,13 +3,21 @@
 This document contains troubleshooting steps for commonly reported issues when using the
 [PowerShell Extension] for Visual Studio Code.
 
-## Script Analysis is Reporting False Errors
+## How do I debug my PowerShell script?
 
-Script analysis is provided by the [PSScriptAnalyzer] project on GitHub.
-Please [open an issue there] if you are getting fault script diagnostics
-(red and green squiggly lines under PowerShell in scripts).
+This topic is best covered in the "Debugging PowerShell Script in Visual Studio Code"
+Scripting Guys blog posts (thanks community!):
 
-## Problems with Syntax Highlighting
+* [Part 1](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/06/debugging-powershell-script-in-visual-studio-code-part-1/)
+* [Part 2](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/13/debugging-powershell-script-in-visual-studio-code-part-2/)
+
+## Script analysis is reporting false errors
+
+Script analysis is provided by the [PSScriptAnalyzer] project on GitHub. If the warning
+message starts with `[PSScriptAnalyzer]` or if you are getting faulty script diagnostics
+(red and green squiggly lines under PowerShell in scripts) please [open an issue there].
+
+## Problems with syntax highlighting
```

Comment:
Ah, this is no longer true

Reply:
PSSA or syntax highlighting?

Reply:
Updated the latter with further clarification, since I'm quite sure script analysis is still PSSA's domain.